requirements.md

```diff
@@ -1,55 +1,52 @@
 
+- **Q:** Are there concerns over the destructive potential of (~12) publishers affecting entries they do not own sufficient to warrent a mitigation approach? **A:** mitigated by having a single team consolidate Solr index metadata, and limited future publishing to the minimum required publishers.
+
+- **Q** Can we streamline the transition by having a single person/team consolidating all the metadata in the Globus Search indexes? **A:** YES, tentatively.
+
 ### ESGF metadata schema
 
-The ESGF dataset metadata schema is based on WCRP specifications and will not change in either Phase I or Phase II.
+The ESGF dataset metadata schema is based on WCRP specifications and will not change ESGF-1.5.
 
-It is expected that the metadata schema will change in Phase II to allow the expression of multiple data nodes (physical storage locations) for a given dataset entry.
+The metadata schema will change to allow the expression of multiple replicas (physical storage locations) for a given dataset entry.
 
-- **Q:** Have we settled on the specific changes mentioned above yet?
+The metadata schema will add these fields to improve the representation of files available from Globus:
 
-- **Q:** What process(es) will we follow to announce the Phase II changes to the ESGF community?
+- New file attributes: collection (string), path (string), type (string), access scheme (string) = "globus", mime type 
 
-### File-level entries
+- **Q:** What process(es) will we follow to announce the metadata changes to the ESGF community?
 
-In Phase I, file-level entries will be included in the Globus Search indices.
+### File-level entries
 
-- **Q:** Could we remove file-level entries in Phase I and replace them with a file manifest (file list including pathnames and checksums for each file in the dataset) in the dataset entry? This would have operational benefits and also simplify clients by reducing the number of entries they must explore to retrieve data.
+File-level entries will be included in the Globus Search index to represent replica locations.
 
-In Phase II, the consolidated index will NOT include file-level entries in the Globus Search index. Clients will not need to inspect additional index entries to learn about individual files in a dataset. Instead, each dataset entry will contain a new file manifest field: a file list including pathnames and checksums for each file in the dataset. Clients can use that manifest to request individual files via HTTP/S. (Clients that prefer bulk transfer can use the base path in the dataset as the source path for transfer requests.)
+- **Q:** Could we remove file-level entries and replace them with a file manifest (file list including pathnames and checksums for each file in the dataset) in the dataset entry? This would have operational benefits and also simplify clients by reducing the number of entries they must explore to retrieve data. **A:** NO, as this change is not necessary and results in more transition effort.
```

Comment:
As you see how it works, you start to grasp how much duplication is in the index and it is tempting to 'fix' it, but I agree to leave things as they are and fix it in NG.